### PR TITLE
Accept env var PCIIDS_LOCAL_PATH to read a local file instead of downloading from GitHub

### PR DESCRIPTION
**Pull request by yeahdongcn**
My user case could be described as follows:

1. My program may run in a network-restricted environment
2. My program may be bundled inside a docker image and deployed into a Kubernetes cluster. I can mount a particular directory from the host to the container with predefined `pci.ids` file

Tests:

```bash
❯ PCIIDS_LOCAL_PATH=pci.ids ./pciids --debug=true 1ed5
DEBUG looking up 1ed5                              
DEBUG using local path pci.ids                     
DEBUG parsing vendor IDs                           
DEBUG parsing PCI IDs                              
DEBUG found 19 result(s)
```

```bash
❯ ./pciids --debug=true 1ed5
DEBUG looking up 1ed5                              
DEBUG downloading https://raw.githubusercontent.com/pciutils/pciids/master/pci.ids 
DEBUG 200 OK                                       
DEBUG parsing vendor IDs                           
DEBUG parsing PCI IDs                              
DEBUG found 19 result(s)   
```

Signed-off-by: Xiaodong Ye <yeahdongcn@gmail.com>

### CODE REVIEW

**Comment by powersj** [comment]
hi again!

Instead of having a mix of environment variables and flags, what do you think about doing this as a flag (e.g. `--file`?

**Comment by yeahdongcn** [comment]
That is doable 🤔

But the side effect is all exported functions need to take an extra parameter. For backward compatibility, this may not be a good idea.

From CLI point of view, adding a flag to horner to the newly added env var should work too.

File: query.go [hunk 1]
```diff
@@ -5,26 +5,56 @@ import (
...
+		}
 	}
 
 	parsedIDs := parse(rawIDs)
 
 	return parsedIDs, nil
 }
 
+func Local(path string) (string, error) {
+	log.Debug("using local path ", path)
```
Comment (powersj) [inline query.go]:
```suggestion
	log.Debug("reading: ", path)
```

**Review by powersj — COMMENTED** [review]
Thought about this some more and I actually have come back around and like how you have this :)

Can you change one debug message and then this is good to go. Thanks again!

File: query.go (hunk 1 above)
Comment (yeahdongcn) [reply query.go]: Updated. Thanks for the review!

**Comment by yeahdongcn** [comment]
Could you please tag a new release? Thanks.

**Comment by powersj** [comment]
Already done! v2.4.0 is out

**Pull request merged**